plugins/ProfessionalServices/Widgets/PromoAbTesting.php

```diff
@@ -9,27 +9,42 @@
 namespace Piwik\Plugins\ProfessionalServices\Widgets;
 
 use Piwik\Container\StaticContainer;
+use Piwik\Piwik;
+use Piwik\View;
 use Piwik\Widget\Widget;
 use Piwik\Widget\WidgetConfig;
 
 class PromoAbTesting extends Widget
 {
+    private const PROMO_PLUGIN_NAME = 'AbTesting';
+
     public static function configure(WidgetConfig $config)
     {
         $config->setCategoryId('ProfessionalServices_PromoAbTesting');
         $config->setSubcategoryId('ProfessionalServices_PromoOverview');
-        $config->setName('ProfessionalServices_PromoAbTestingOverview');
         $config->setIsNotWidgetizable();
 
         $promoWidgetApplicable = StaticContainer::get('Piwik\Plugins\ProfessionalServices\PromoWidgetApplicable');
 
-        $isEnabled = $promoWidgetApplicable->check('AbTesting');
-        $isEnabled = false;
+        $isEnabled = $promoWidgetApplicable->check(self::PROMO_PLUGIN_NAME);
         $config->setIsEnabled($isEnabled);
     }
 
     public function render()
     {
-        return 'content';
+        $marketplacePlugins = StaticContainer::get('Piwik\Plugins\Marketplace\Plugins');
+        $pluginInfo = $marketplacePlugins->getPluginInfo(self::PROMO_PLUGIN_NAME);
```

Comment:
TL;DR: This ultimately extends the time to see the promo as it does a marketplace API call.

I think it's the right way to go about getting any plugin related info like the link to its detail page on plugins.matomo.org or its display name, however, it slows it down.

We can pass hardcode the information here for the 2-3 fields we use for each plugin and display the ad/promo content faster. The risk is that information might become out of sync with the marketplace around the plugin display name, the URL on which the user can purchase purchase it etc.

What are the opinions on this, @Stan-vw @caddoo @bx80?

Reply:
I would say make it static for now, if we then have a problem with syncing (how often do those values really change?) then we go for the slower approach

Reply:
How are the current links in the marketplace coded, e.g. are they hardcoded or are they pulled from the API?
I suggest we stay consistent, rather than reinventing the wheel.

Reply:
Another problem with static data would be the delay with people deploying new versions on-premise, so if static plugin data ever did change then an invalid name/link could be shown for months or years before it can be fixed by a new version.

 Also worth noting that [plugin data should be cached](https://github.com/matomo-org/matomo/blob/2c878f46ec3e0d270e4ae4231ff31721dacc7dd9/plugins/Marketplace/Api/Client.php#L266) after the first request so the delay should be much less for subsequent requests.

Reply:
Seems like we're on the fence with this so I'd leave it as is (it's safer) and we can always hardcode it later if we need to.